cache-config/testing/docker/docker-compose-ats-build.yml

```diff
@@ -27,11 +27,14 @@ services:
 
   trafficserver_build:
     environment:
-      - ATS_VERSION=8.1.x
+      - CJOSE_URL=https://github.com/cisco/cjose
+      - CJOSE_TAG=latest
+      - JANSSON_URL=https://github.com/akheron/jansson
+      - JANSSON_TAG=v2.11
+      - OPENSSL_URL=https://github.com/openssl/openssl
+      - OPENSSL_TAG=OpenSSL_1_1_1
     build:
       context: ../../..
       dockerfile: cache-config/testing/docker/trafficserver/Dockerfile
-      args:
-        RHEL_VERSION: ${RHEL_VERSION:-7}
```

Comment:
Why remove the `RHEL_VERSION` build arg? Without it, `RHEL_VERSION` is not inherited from the environment, which is how the CDN in a Box Makefile passes the `RHEL_VERSION` build arg to Dockerfiles.

Reply:
@zrhoffman is this something new, docker-compose doesn't support Parameter expansion
`docker-compose -f docker-compose-ats-build.yml build trafficserver_build
ERROR: Invalid interpolation format for "build" option in service "trafficserver_build": "${RHEL_VERSION:-7}"`